packages/components/src/components/map-marker/map-marker.stories.ts

```diff
@@ -28,155 +39,82 @@ export const Default = {
 };
 
 /**
- * The marker in its 'Cluster' variant in all possible states.
+ * Use the `variant` attribute to change the appearance of the marker.
+ *
+ * - `main`: The default variant.
+ * - `place`: A variant that is used to show a location on the map.
+ * - `cluster`: A variant that is used to show multiple locations on the map.
  */
-export const VariantCluster = {
-  name: 'Variant Cluster',
-  parameters: { controls: { exclude: ['state'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'state', values: ['default', 'hover'] }
-      },
-      args,
-      constants: [
-        {
-          type: 'slot',
-          name: 'default',
-          value: '8'
-        },
-        {
-          type: 'attribute',
-          name: 'variant',
-          value: 'cluster'
-        }
-      ]
-    });
-  }
+export const Variant = {
+  render: () => html`
+    <div class="flex items-center gap-12">
+      <sd-map-marker></sd-map-marker>
+      <sd-map-marker variant="place">
+        <sd-icon name="content/image" color="primary"></sd-icon>
+      </sd-map-marker>
+      <sd-map-marker variant="cluster"> 8 </sd-map-marker>
+    </div>
+  `
 };
 
 /**
- * The marker in its 'Main' variant in all possible states.
+ * Use the `state` attribute to change the appearance of the marker.
+ *
+ * - `default`: The default state.
+ * - `hover`: The state when the marker is hovered.
+ * - `active`: The state when the marker is active.
  */
```

Comment:
for the sample use 88 instead of 8 on both variants

Reply:
Updated